.gitignore

```diff
@@ -5,4 +5,4 @@ archive/*
 build/*
 downloads/*
 diff/*
-dist/*
+Python-*-*-support.*.tar.gz
```

Comment:
I also reverted the change to Python.patch that adds -fembed-bitcode now that Python build is done off the root Makefile.